test/e2e/gardener/shoot/create_and_delete_hibernated.go

```diff
@@ -40,9 +43,17 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 		Expect(f.CreateShootAndWaitForCreation(ctx, false)).To(Succeed())
 		f.Verify()
 
+		verifyNoPodsRunning(f)
```

Comment:
Can we do this in https://github.com/gardener/gardener/blob/master/test/e2e/gardener/shoot/create_hibernate_wakeup_delete.go as well, please?